aggregate/central-site/pom.xml

```diff
@@ -8,7 +8,7 @@
     <version>4.3.0-SNAPSHOT</version>
   </parent>
   <artifactId>org.jboss.tools.site.central</artifactId>
-  <name>JBoss Tools - Central</name>
+  <name>JBoss Central</name>
```

Comment:
Because the same site (all the same contents) is built for both JBT and JBDS. So in both cases, it's just JBoss Central.